crates/sui-framework/deps/move-stdlib/sources/ascii.move

```diff
@@ -1,15 +1,11 @@
-// Copyright (c) 2022, Mysten Labs, Inc.
-// SPDX-License-Identifier: Apache-2.0
-
 /// The `ASCII` module defines basic string and char newtypes in Move that verify
 /// that characters are valid ASCII, and that strings consist of only valid ASCII characters.
 module std::ascii {
     use std::vector;
-    use std::errors;
     use std::option::{Self, Option};
 
     /// An invalid ASCII character was encountered when creating an ASCII string.
-    const EINVALID_ASCII_CHARACTER: u64 = 0;
+    const EINVALID_ASCII_CHARACTER: u64 = 0x10000;
```

Comment:
LGTM, but why these specific numbers?

Reply:
Just copying what is in upstream Move. I think these correspond to codes in errors.move